test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/DefaultTagHelperContentTest.cs

```diff
@@ -360,6 +457,37 @@ public void Fluent_SetContent_Append_WritesExpectedContent()
         }
 
         [Fact]
+        public void Fluent_SetContent_AppendFormat_WritesExpectedContent()
+        {
+            // Arrange
+            var tagHelperContent = new DefaultTagHelperContent();
+            var expected = "Hello World test";
```

Comment:
no, it would be confirming that the (public) enumeration keeps the values separate